docs/pages/includes/database-access/db-audit-events.mdx

```diff
@@ -0,0 +1,38 @@
+Database audit events will appear in the Audit Log including database sessions starting,
+query activity and ending. After the session has been uploaded you can retrieve the
+session activity with `tsh play`.
+
+```code
+# Database session id will be in a UUID format (ex: 307b49d6-56c7-4d20-8cf0-5bc5348a7101)
+$ tsh play --format json <Var name="database.session" />
+```
+
+Example output:
+```json
+    {
+        "cluster_name": "teleport.example.com",
+        "code": "TDB02I",
+        "db_name": "example",
+        "db_origin": "dynamic",
+        "db_protocol": "postgres",
+        "db_query": "select * from sample;",
+        "db_roles": [
+            "access"
+        ],
+        "db_service": "example",
+        "db_type": "rds",
+        "db_uri": "databases-1.us-east-1.rds.amazonaws.com:5432",
+        "db_user": "alice",
+        "ei": 2,
+        "event": "db.session.query",
+        "sid": "307b49d6-56c7-4d20-8cf0-5bc5348a7101",
+        "success": true,
+        "time": "2023-10-06T10:58:32.88Z",
+        "uid": "a649d925-9dac-44cc-bd04-4387c295580f",
+        "user": "alice"
+    }
+```
+
+The Audit Log is viewable in Activity under Management in the Web UI for users
+with permission to the `event` resources. Database sessions do not appear
+under the recorded sessions listing.
```

Comment:
Should we say "on the session recordings" page instead of "the recorded sessions listing"

Reply:
Yes, that is more clear.